scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1418,21 +1418,15 @@
 
 (defn create-mesos-scheduler
   "Creates the mesos scheduler which processes status updates asynchronously but in order of receipt."
-  [configured-framework-id gpu-enabled? conn heartbeat-ch pool->fenzo pool->offers-chan match-trigger-chan
+  [gpu-enabled? conn heartbeat-ch pool->fenzo pool->offers-chan match-trigger-chan
    handle-exit-code handle-progress-message sandbox-syncer-state]
-  (let [sync-agent-sandboxes-fn #(sandbox/sync-agent-sandboxes sandbox-syncer-state configured-framework-id %1 %2)
+  (let [sync-agent-sandboxes-fn #(sandbox/sync-agent-sandboxes sandbox-syncer-state @cook.config/framework-id-atom %1 %2)
         message-handlers {:handle-exit-code handle-exit-code
                           :handle-progress-message handle-progress-message}]
     (mesos/scheduler
       (registered
         [this driver framework-id master-info]
         (log/info "Registered with mesos with framework-id " framework-id)
-        (let [value (-> framework-id mesomatic.types/pb->data :value)]
```

Comment:
Why was this removed?

Reply:
It seemed unnecessary. We get the driver via code around here:

https://github.com/scrosby/Cook/blob/outgoing/cleanup-framework-id/scheduler/src/cook/mesos.clj#L197
create-datomic-scheduler calls create-mesos-scheduler.

Notice that we it what framework id we are. So if we ever get a registration callback for another framework-id, something is really screwed up; its a bug in our implementation/initialization. The driver atom is initialized about 20 lines further down.

We could retain this code and have it refer to the global variable, but I don't see the point, or the value.

Reply:
I think the point of this is to make sure there's not something really screwed up :)

It seems pretty harmless to me as a sanity check.

Reply:
Left in.